### PR TITLE
Adding sorting column in the query builder and results

### PR DESCRIPTION
**Pull request by ddelpiano**
- added the variable sorterColumn that drive the sorting of the query results.

### CODE REVIEW

**Comment by ddelpiano** [comment]
this needs [this](https://github.com/openworm/geppetto-client/pull/97) to work.

**Comment by ddelpiano** [comment]
@Robbie1977 @gidili this is ready for the review and goes together with [this PR](https://github.com/openworm/geppetto-client/pull/97).

Brief explanation:

The sorting is driven by the array defined in ./components/configuration/queryBuilderConfiguration.js called sorterColumns, as per below:

```
var sorterColumns = [
  {
    column: "score",
    order: "DESC"
  },
  {
    column: "name",
    order: "ASC"
  },
  {
    column: "images",
    order: "DESC"
  },
];
```

This array  of objects will be looped in order and compared with the results columns present each time we run a query, if the column header and the column parameter defined in this array match that will drive the sorting using the favourite order defined as well in order. 
So to sum up, the column parameter as to match whatever we place in the field columnName in the queryResultsColMeta (everything is lowered so upper or lower case does not matter).

The order is a different story, initially I was using what Griddle is providing you with the param "initialSortAscending", however this does not work so I had to re-think a little bit this and I ended up hacking this using the sortDirectionCycle to get the order ASC or DESC to work, the rest is quite explicative from the code itself.
